### PR TITLE
=doc Fix typos of indefinite articles in docs

### PR DESCRIPTION
**Pull request by naferx**

### CODE REVIEW

File: akka-docs/rst/java/camel.rst [hunk 1]
```diff
@@ -281,7 +281,7 @@ Actors may also use a Camel `ProducerTemplate`_ for producing messages to endpoi
 
 .. includecode:: code/docs/camel/MyActor.java#ProducerTemplate
 
-For initiating a a two-way message exchange, one of the
```
Comment (ktoso) [inline akka-docs/rst/java/camel.rst]: those are always a mystery to me... thanks!

Comment (naferx) [inline akka-docs/rst/java/camel.rst]: In the IntelliJ IDEA find dialog, this regular expression 
`\b(\w+)\s+\1\b` 
catches all consecutive duplicate words in `.rst` files.

Comment (ktoso) [inline akka-docs/rst/java/camel.rst]: nice, good hint. Thanks!

**Comment by johanandren** [comment]
LGTM

**Comment by ktoso** [comment]
Thanks @naferx !

**Pull request merged**